doc/articles/getting-started/getting-started-tests.md

```diff
@@ -0,0 +1,59 @@
+---
+uid: Uno.Authoring.Tests
+---
+# How-To: Creating an application with tests
+ 
+This tutorial will walk through how to create an Uno application with tests.
+
+> [!NOTE]
+> Make sure to setup your environment first by [following our instructions](xref:Uno.GetStarted.vs2022).
```

Comment:
I would love to see reference to this [(Wizard doc regarding tests)](https://platform.uno/docs/articles/getting-started/wizard/using-wizard.html#10-testing) on this page, maybe as note, so that people can know what they need to do to create a solution with tests.

@agneszitte @vatsashah45

Reply:
Nick already agreed on this here https://github.com/unoplatform/uno/pull/16314#discussion_r1567685830